paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -551,7 +560,6 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
         } else {
             createUSBankAccountConfiguration()
         }
-        _analyticsEvent.tryEmit(AnalyticsEvent.Started)
```

Comment:
Moving this from `collectBankAccountForIntent` to `collectBankAccount` so that all calls are covered.